Taskfile.R.yaml

```diff
@@ -23,7 +23,8 @@ tasks:
     cmds:
       - task copy-root-files LANGUAGE=R IDENTIFIER={{.IDENTIFIER}} NBL_LANGUAGE_VERSION={{.NBL_R_VERSION}}
       - task copy-language-files LANGUAGE=R IDENTIFIER={{.IDENTIFIER}} NBL_LANGUAGE_VERSION={{.NBL_R_VERSION}}
-
+      - cp R/noteable/.Rprofile R/noteable/{{.NBL_R_VERSION}}/.Rprofile
```

Comment:
These shouldn't be needed for the base build; may need them for a follow-on build for common `datascience`-like builds though

Reply:
I'll probably end up moving some of these before the CI work is completed. These commands were only executed by `task` when running the `noteable:build`, but as that will all be in `docker buildx bake`, I need to copy them before hand